hibernate-core/src/main/java/org/hibernate/engine/spi/EntityKey.java

```diff
@@ -29,7 +30,9 @@
  * @author Gavin King
  * @author Sanne Grinovero
  */
-public final class EntityKey implements Serializable {
+public final class EntityKey implements Serializable, SerializableKey {
+
+	private static final long serialVersionUID = 776292106989119341L;
```

Comment:
please don't add any `serialVersionUID`, we intentionally decided it's best to not have them.  It's a long story ... you can probably find the discussion on the mailing list archives.

Reply:
The discourse one if for users and usage questions. Developers use http://lists.jboss.org/pipermail/hibernate-dev/

See also http://hibernate.org/community/ : you're welcome to join.

Reply:
I've just joined! Can you say when this discussion took place? The archive is long.

PS: Sorry for deviating the PR discussion, but since you mentioned it...

Reply:
lol, looks like that was more than 10y ago :)
I still remembered it..

http://lists.jboss.org/pipermail/hibernate-dev/2008-May/003097.html

FYI you can search using this service, based on Elasticsearch:
 - http://search.jboss.org/#!q=serialVersionUID&project=hibernate

Reply:
Thank you so much! I just saw a few formatting undos I missed in my last commits. I'll be tackling that review today